### PR TITLE
Lps 53951

### PR DESCRIPTION
**Pull request by arbetts**

### CODE REVIEW

File: portal-impl/src/com/liferay/portlet/dynamicdatamapping/service/impl/DDMTemplateLocalServiceImpl.java [hunk 1]
```diff
@@ -1409,14 +1412,13 @@ protected String formatScript(String type, String language, String script)
...
 			}
-			catch (Exception e) {
-				throw new TemplateScriptException();
+			catch (PortalException pe) {
+				throw new TemplateScriptException(pe);
 			}
 		}
 
-		return script;
+		return DDMXMLUtil.formatXML(script);
```
Comment (jonathanmccann) [inline portal-impl/src/com/liferay/portlet/dynamicdatamapping/service/impl/DDMTemplateLocalServiceImpl.java]: I think that this should still be in a try/catch and this is also changing the flow of logic (now it always formats it instead of when the condition is true). Why did you move it out?

Comment (arbetts) [inline portal-impl/src/com/liferay/portlet/dynamicdatamapping/service/impl/DDMTemplateLocalServiceImpl.java]: you're right about the if check, it should be inside of it... but the systemException is a runtime exception in trunk so we don't need to catch it.

**Comment by jonathanmccann** [comment]
Reviewed and sent to Marcellus for further review: https://github.com/marcellustavares/liferay-portal/pull/426
